Sources/NIO/SelectableEventLoop.swift

```diff
@@ -146,13 +146,28 @@ internal final class SelectableEventLoop: EventLoop {
         }
     }
 
+    public var safeToExecute: Bool {
```

Comment:
I'm inclined to make this `internal`: the type is `internal` so the access control of this computed property doesn't _really_ matter, but I don't think we'd normally want people to be able to use it.

Additionally, we'd normally want it to be `private` but we want access to it for testing purposes. The convention in NIO to annotate this is to use:

```swift
/* private but internal */ var _safeToExecute: Bool
```

This comments why the access modifier is higher than we want, and also prefixes it with an underscore to indicate to other parts of the code that we probably shouldn't use it.

Reply:
I agree